giraffe/src/utils/getTicks.ts

```diff
@@ -29,19 +26,23 @@ const getNumTicks = (
   charLength: number
 ): number => {
   const sampleTickWidth = sampleTick.length * charLength
-  const numTicks = Math.round((length / sampleTickWidth) * TICK_DENSITY)
-
-  return Math.max(numTicks, 2)
+  return Math.round(length / sampleTickWidth)
 }
 
 const getTimeTicks = (
   [d0, d1]: number[],
   length: number,
   numTicks: number
 ): number[] => {
-  return scaleUtc()
+  const results = scaleUtc()
     .domain([d0, d1])
     .range([0, length])
     .ticks(numTicks)
     .map(d => d.getTime())
+  if (results.length > numTicks) {
```

Comment:
non blocker: something that would be super helpful would be a comment explaining what it means or represents when `results.length > numTicks` - i can't quite figure it out from local context without making an educated guess or looking up what `scaleUtc` does.

Reply:
@hoorayimhelping do you mean to move this comment outside of the if?

https://github.com/influxdata/giraffe/pull/110/files#diff-325e130ef8529ac0832abade9a2f50bdR43-R44

Reply:
i guess it's this: i see that we're sending the results array up to `numTicks` to appease d3, i just don't understand why `numTicks` would be greater than `results.length` or what it really means for the state of the app